compiler/runtime.go

```diff
@@ -471,8 +479,7 @@ func (e *memberEvaluator) Eval(scope Scope) (values.Value, error) {
 		return nil, err
 	}
 	v, _ := o.Object().Get(e.property)
-	// TODO(nathanielc): Fix this poly type issue
-	//values.CheckKind(v.PolyType().Nature(), e.t.Nature())
+	values.CheckKind(v.Type().Nature(), e.t.Nature())
```

Comment:
Does member expression need to use the poly Type as well? If not why? The issue I ran into was `http.post` which was a polymorphic member expression.